Source/Scene/Scene.js

```diff
@@ -1659,6 +1716,108 @@ define([
         }
     }
 
+    function insertShadowCommands(scene, commandList, insertAll, shadowMap, shadowPassCommands) {
```

Comment:
For the roadmap, do you think it is worthwhile to consider doing all the culling (view frustum and shadow maps) at once instead of distributing it between createPVS and here?